app/serializers/referee_serializer.rb

```diff
@@ -36,7 +36,6 @@ class RefereeSerializer
   attributes :first_name,
              :last_name,
              :bio,
-             :email,
```

Comment:
This was deeply unsafe and mostly useless. If refs need to change their email they can reach out to the IQA. No one has requested this functionality so far.